mod-invoice-storage/schemas/invoice.json

```diff
@@ -137,7 +155,7 @@
     "approvalDate",
```

Comment:
I know this change is not part of this story, but I think approval date should not be a required field in schema as it's based on status of invoice